degruyter-harvard.csl

```diff
@@ -0,0 +1,338 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>De Gruyter Harvard</title>
+    <id>http://www.zotero.org/styles/degruyter-harvard</id>
+    <link href="http://www.zotero.org/styles/degruyter-harvard" rel="self"/>
+    <author>
```

Comment:
Please remove all of these authors and just have the author that made the current new style (the other authors are acknowledged by the template link you will add)